.github/workflows/ci.yaml

```diff
@@ -138,6 +138,10 @@ jobs:
             test: { name: "apertis", case: "apertis" }
           - backend: kvm
             test: { name: "kali", case: "kali" }
+          - backend: kvm
+            test: { name: "apertis v2022", case: "apertis", variables: "-t suite:v2022" }
```

Comment:
2022 is just went EOL ; so lets not have it there; if you want something bullseye based v2023 is the right one